src/vit-servicing-station/vit-servicing-station-lib/src/db/mod.rs

```diff
@@ -28,12 +68,18 @@ WHERE
 ";
 
 pub fn load_db_connection_pool(db_url: &str) -> Result<DbConnectionPool, Error> {
-    let manager = ConnectionManager::<SqliteConnection>::new(db_url);
-    let pool = Pool::builder().build(manager)?;
+    if db_url.starts_with("postgres://") {
+        Ok(DbConnectionPool::Postgres(
+            Pool::builder().build(ConnectionManager::new(db_url))?,
+        ))
+    } else {
+        let manager = ConnectionManager::<SqliteConnection>::new(db_url);
+        let pool = Pool::builder().build(manager)?;
 
-    // test db connection or bubble up error
-    let conn = pool.get()?;
-    conn.execute(TEST_CONN_QUERY)?;
+        // test db connection or bubble up error
+        let conn = pool.get()?;
+        conn.execute(SQLITE_TEST_CONN_QUERY)?;
```

Comment:
That code was already there. I didn't check if this is necessary. For Postgres an error is thrown if establishing the connection fails. I'll check this. Maybe it can be removed.

I tried to change the SQLite part of the code as little as possible to get vit-ss working with Postgres that's why there are things like this.